targets/postman/make.js

```diff
@@ -150,6 +150,17 @@ function getPostmanHeaderV2(apiCall) {
                 "value": "{{EntityToken}}"
             }
         ])
+    else if (apiCall.auth === "AadToken")
```

Comment:
Was this a new API we were hitting this on? Can you share more details?

Reply:
'SetTitleInternalDataAndOverrides', apparently has a .auth set to AadToken, so we may need to also add another group of 

"key": "X-AadToken"
"value": "{{AadToken}}"

But I just want to confirm that it would then require folks to add an AadToken manually to their postman collection